app/models/User.py

```diff
@@ -117,7 +118,8 @@ def get_profile_picture(self, size=50):
         if self.image:
             return self.image.url()
         if not self.image_url:
-            return 'https://lh6.googleusercontent.com/-K9HZ5Z5vOU8/AAAAAAAAAAI/AAAAAAAAAAA/yRoMtBSXoxQ/s48-c/photo.jpg'
+            return ('https://lh6.googleusercontent.com/-K9HZ5Z5vOU8/'
```

Comment:
is it possible to have a better URL here?
(not needed for this PR, but could be an issue for later)